engine/src/main/java/io/zeebe/engine/processing/bpmn/container/CallActivityProcessor.java

```diff
@@ -132,7 +137,7 @@ public void onChildActivating(
       final BpmnElementContext childContext) {}
 
   @Override
-  public void onChildCompleted(
+  public void beforeExecutionPathCompleting(
```

Comment:
In ProcessProcessor, you've moved the current implementation of onChildCompleted into the after method. Is it necessary here to move it into the before? Otherwise, I'd prefer consistency and start by having this code in the method that matches most with the completed state (i.e. after)

Reply:
Yes, it is because of `stateBehavior.propagateTemporaryVariables(childContext, callActivityContext);`